src/common/components/Profile/ProfileCard.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
```

Comment:
@reacteer add the component to storybook

Reply:
@dbizhev , I am not familiar with storybook. Can we do that after merge?

Reply:
@reacteer I'll add the profile card component to the storybook on the current branch.